fluent-package/apt/systemd-test/update-to-next-version-service-status.sh

```diff
@@ -42,10 +42,13 @@ else
     (! systemctl is-enabled fluentd)
 fi
 
-if [ "$status_before_update" = active ] || [ "$enabled_before_update" = enabled ] ; then
-    # The service should restart automatically after update
+if [ "$status_before_update" = active ]; then
+    # The service should NOT restart automatically after update
```

Comment:
I'd like to more comments like this to clarify.

```suggestion
    # The service should NOT restart automatically after update
    # (The process before update should continue to run)
```